disperser/apiserver/server.go

```diff
@@ -93,12 +93,12 @@ func (s *DispersalServer) DisperseBlobAuthenticated(stream pb.Disperser_Disperse
 	// Process disperse_request
 	in, err := stream.Recv()
 	if err != nil {
-		return fmt.Errorf("error receiving next message: %v", err)
+		return common.NewInvalidArgError(fmt.Sprintf("error receiving next message: %v", err))
```

Comment:
nit: Is this really due to invalid argument? Feel like stream.Revc() breaking could be due to pod dying in the middle or something like that.

Reply:
good point, this may be not a clear cut whether it's client side or server side issue, as it may be caused by various reasons, like network connection, malformed message as well as server side resource issue etc.
I think overall it may fit better as client-side, as this is still at the gate of the system at this point.